utils/calc-stack-height/src/main.rs

```diff
@@ -108,13 +119,16 @@ fn main() -> anyhow::Result<()> {
             |module| schedule.rules(module),
             Some(mid),
             schedule.limits.data_segments_amount.into(),
-            schedule.limits.table_number.into(),
         )
-        .map_err(|e| anyhow!("{e}"))?;
-
-        let module = Module::new(&store, code.code())?;
-        let instance = Instance::new(&mut store, &module, &imports)?;
-        let init = instance.exports.get_function("init")?;
+        .context("Code error")?;
+
+        let module = Module::new(&store, code.code()).context("Failed to create module")?;
+        let instance =
+            Instance::new(&mut store, &module, &imports).context("Failed to instantiate module")?;
+        let init = instance
+            .exports
+            .get_function("init")
+            .context("Failed to get `init` function export")?;
```

Comment:
dont we plan unifying each .context usage with a style of lowercase "failed to ..."